src/phoenix/db/migrate.py

```diff
@@ -1,10 +1,31 @@
 import logging
+from contextlib import contextmanager
 from pathlib import Path
+from typing import Any, Generator
 
 from alembic import command
 from alembic.config import Config
 from sqlalchemy import URL
 
+
+@contextmanager
+def using_log_level(new_level: int, logger_names: list[str]) -> "Generator[Any, Any, Any]":
```

Comment:
```suggestion
def using_log_level(new_level: int, logger_names: List[str]) -> Iterator[Any]:
```

I don't think `list` works as a generic type on the lowest versions of Python we support.

Reply:
Ah yeah good catch